src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -110,9 +110,12 @@ namespace
         ParamIdAdaptiveTileNoiseThreshold               = 44,
         ParamIdBackgroundAlphaValue                     = 15,
         ParamIdNoiseSeed                                = 76,
+        ParamIdEnableNoiseSeed                          = 77,
```

Comment:
`ParamIdEnableNoiseSeed` is not quite accurate: it should be something like `ParamIdEnablePerFrameNoiseSeedVariation`.

Reply:
ok